server/Cargo.toml

```diff
@@ -56,6 +57,7 @@ static-toml = "1.2.0"
 strip-ansi-escapes = "0.2.0"
 strum = { version = "0.26.2", features = ["derive"] }
 sysinfo = "0.32.0"
+tar = "0.4.42"
```

Comment:
Looks like this dependency is not needed (at least according to `cargo machete`).

Reply:
yep, you're right, removed

Reply:
@haze518 looks good now. We've just merged another PR, so please update the server version to `0.4.71` :)

Reply:
@spetz done